README.md

```diff
@@ -348,7 +347,7 @@ Specifically there are 2 ways of doing so: Pure Python and via a Runner
 
 ```
 # Execute
-python3 torchchat.py generate llama3.1 --device cpu --pte-path llama3.1.pte --prompt "Hello my name is"
+python3 torchchat.py generate llama3.1 --pte-path llama3.1.pte --prompt "Hello my name is"
```

Comment:
PTE always defaults to CPU, no explicit arg needed

Reply:
is there any warnings in our codebase to tell user that?

Reply:
Yup, the logs will say it falls back to CPU (even if they ask for GPU)